schema/validator.go

```diff
@@ -22,8 +22,8 @@ import (
 	"io/ioutil"
 	"regexp"
 
+	v1 "github.com/opencontainerd/image-spec/specs-go/v1"
```

Comment:
```suggestion
	v1 "github.com/opencontainers/image-spec/specs-go/v1"
```

😉

Reply:
I think if you actually accept this suggestion through the UI it might break DCO, so caveat emptor.

Reply:
Love that muscle memory 😆 
It was also in my $GOPATH.